e2e-tests/package.json

```diff
@@ -8,7 +8,7 @@
   "devDependencies": {
     "@matterlabs/eslint-config-vue": "^1.1.4",
     "@matterlabs/hardhat-zksync-deploy": "^0.6.5",
-    "@matterlabs/hardhat-zksync-solc": "^0.4.2",
+    "@matterlabs/hardhat-zksync-solc": "^1.2.5",
```

Comment:
We should just update fully here and use  `@matterlabs/hardhat-zksync` ? Thoughts?

Reply:
Yeah, let's do it